<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerView</class>
 <widget class="QWidget" name="ServerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="searchHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblSearchButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>20</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_search.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
padding-left:0px;
color: #818083;
border-color:transparent;
}</string>
    </property>
    <property name="placeholderText">
     <string>Search locations...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSearchLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_search_gray.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>180</width>
     <height>270</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeWidget::item{
height: 50px;
border: 0px;  
padding: 0px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="uniformRowHeights">
    <bool>false</bool>
   </property>
   <property name="animated">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>526</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>526</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>526</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>530</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnAll">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background:#FACA7F;
border-radius: 10px;
color:#000000;
}</string>
        </property>
        <property name="text">
         <string>ALL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFav">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
        </property>
        <property name="text">
         <string>FAVORITE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLastUsed">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
        </property>
        <property name="text">
         <string>LAST USED</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHighSpeed">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
        </property>
        <property name="text">
         <string> HIGH SPEED</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStreaming">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
        </property>
        <property name="text">
         <string>STREAMING</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdBlocker">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
        </property>
        <property name="text">
         <string>AD BLOCKER</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
